mito-ai/mito_ai/gemini_client.py

```diff
@@ -19,19 +20,41 @@ def __init__(self, api_key: Optional[str], model: str):
         if api_key:
             self.client = genai.Client(api_key=api_key)
 
+    def convert_messages_for_gemini(self, messages: List[Dict[str, Any]]) -> Sequence[Content]:
```

Comment:
Instead of returning a Sequence, can we just return a List so we don't need to cast it later?